src/components/AudioDetail.vue

```diff
@@ -307,6 +318,65 @@ export default defineComponent({
       () => store.state.nowPlayingContinuously
     );
 
+    const pronunciationByPhrase = computed(() => {
+      let textArray: Array<string> = [];
+      accentPhrases.value?.forEach((accentPhrase) => {
+        let textString = "";
+        accentPhrase.moras.forEach((mora) => {
+          textString += mora.text;
+        });
+        if (accentPhrase.pauseMora) {
+          textString += "、";
+        }
+        textArray.push(textString);
+      });
+      return textArray;
+    });
+
+    const handleChangePronounce = (
+      newPronunciation: string,
+      phraseIndex: number
+    ) => {
+      let popUntilPause = false;
+      newPronunciation = newPronunciation.replace(",", "、");
+      if (
+        newPronunciation.slice(-1) == "、" &&
+        accentPhrases.value!.length - 1 != phraseIndex
+      ) {
+        newPronunciation += pronunciationByPhrase.value[phraseIndex + 1];
+        popUntilPause = true;
+      }
+      store.dispatch(FETCH_SINGLE_ACCENT_PHRASE, {
+        audioKey: activeAudioKey.value,
+        newPronunciation,
+        accentPhraseIndex: phraseIndex,
+        popUntilPause,
+      });
+    };
+
+    const hoveredPhraseIndex = ref<number | undefined>(undefined);
+
+    const handleHoverText = (phraseIndex: number, isOver: boolean) => {
+      if (isOver) {
+        hoveredPhraseIndex.value = phraseIndex;
+      } else {
+        hoveredPhraseIndex.value = undefined;
+      }
+    };
+
+    const getHoveredClass = (accentPhraseIndex: number) => {
+      if (selectedDetail.value != "accent") {
+        return "text-cell";
+      }
+      if (!uiLocked.value) {
+        if (accentPhraseIndex === hoveredPhraseIndex.value) {
+          return "text-cell-hovered";
+        } else {
+          return "text-cell";
+        }
+      }
```

Comment:
This may be simpler:
```js
if(selectedDetail.value == "accent" && !uiLocked.value && accentPhraseIndex === hoveredPhraseIndex.value) {
  return "text-cell-hovered";
} else {
  return "text-cell";
}
```